test/exports.js

```diff
@@ -13,3 +13,4 @@ exports["to_ascii"] = to_ascii;
 exports["tokenizer"] = tokenizer;
 exports["TreeTransformer"] = TreeTransformer;
 exports["TreeWalker"] = TreeWalker;
+exports["walk_body"] = walk_body;
```

Comment:
This probably should either be a public export, or a method on TreeWalker. Good enough in test for now.

Reply:
It's just a helper function that can be replicated, unlike `List` which is irreplaceable.

In fact, there should be no `body instanceof AST_Statement` within `uglify-js`, so I'm going to nuke that for a performance bump.

Reply:
You mean removing `AST_Statement` from `uglify-js` altogether?

Reply:
Not that extreme &minus; just removing that `instanceof` clause since it's unreachable. 😅